README.md

```diff
@@ -19,21 +19,25 @@ $ npx tram-one-express app-name
 This will create a directory in your current directory with the name that you pass into tram-one-express.
 
 ## What's in the box
-Below are a list of files and directories that are created after running tram-one-express
+The scaffolded project includes the build tooling, tests, and a sample of components and hooks that use the Tram-One API.
+Below are a list of files and directories that are created after running tram-one-express.
 ```
 app-name/
 ├── src
-│   ├── components
-│   │   └── ColorHeader
-│   │       ├── ColorHeader.js
-│   │       ├── ColorHeader.scss
-│   │       ├── ColorHeader.test.js
-│   │       └── index.js
+│   ├── app-header.js
+│   ├── app-summary.js
+│   ├── app-task-description.js
+│   ├── app-task-item.js
+│   ├── app-task-list.js
 │   ├── index.js
-│   └── styles.css
-├── .gitignore
```

Comment:
Why was this removed?

Reply:
mistake in how `tree` command was used, it actually is laid down, good catch!